tests/acceptance/test_onboarding.py

```diff
@@ -43,7 +43,7 @@ def test_onboarding(self, generate_api_key):
         assert project.platform == 'javascript-angular'
 
         self.browser.snapshot(name='onboarding-configure-project')
-        self.browser.click('.btn-primary')
+        self.browser.click('button[aria-label="All done!"]')
```

Comment:
I would add a `data-test-id` property to the `Button` instead of using the aria-label.